WalletWasabi.Fluent/Views/Wallets/Receive/ReceiveView.axaml

```diff
@@ -4,38 +4,36 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:c="using:WalletWasabi.Fluent.Controls"
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
-             xmlns:behaviors="using:WalletWasabi.Fluent.Behaviors"
-             xmlns:i="using:Avalonia.Xaml.Interactivity"
+             xmlns:behaviors="clr-namespace:WalletWasabi.Fluent.Behaviors"
+             xmlns:i="clr-namespace:Avalonia.Xaml.Interactivity;assembly=Avalonia.Xaml.Interactivity"
              xmlns:receive="clr-namespace:WalletWasabi.Fluent.ViewModels.Wallets.Receive"
              x:DataType="receive:ReceiveViewModel"
              x:CompileBindings="True"
              x:Class="WalletWasabi.Fluent.Views.Wallets.Receive.ReceiveView">
-  <c:ContentArea Title="{Binding Title}"
-                 EnableNext="True" NextContent="Continue"
-                 EnableCancel="{Binding EnableCancel}"
-                 EnableBack="{Binding EnableBack}">
-    <DockPanel>
+  <c:AdorningContentControl RevealMode="Manual" IsAdornmentVisible="True">
+    <c:ContentArea Title="{Binding Title}"
+                   Caption="In order for Wasabi to best protect your privacy enter some labels to keep track of who knows about this receive address."
```

Comment:
```suggestion
                   Caption="Enter labels to keep track of who knows about this receive address, in order for Wasabi to best protect your privacy."
```

Write the important thing first. But this can probably be written much more beautiful.